src/gwt/src/org/rstudio/studio/client/projects/ui/prefs/ProjectGeneralPreferencesPane.java

```diff
@@ -77,6 +83,25 @@ protected void initialize(RProjectOptions options)
       alwaysSaveHistory_.setSelectedIndex(config.getAlwaysSaveHistory());
       tutorialPath_ = config.getTutorialPath();
       rVersion_ = config.getRVersion();
+      
+      // check or uncheck the checkbox for child processes based on the configuration value
+      // if default is specified, we need to use the current session setting
+      // otherwise, yes or no indicate the check state exactly
+      int quitChildProcessesOnExit = config.getQuitChildProcessesOnExit();
+      boolean quitChildProcessesChecked = sessionInfo_.quitChildProcessesOnExit();
+      
+      switch (quitChildProcessesOnExit)
+      {
+      case 1:
```

Comment:
We usually use named constants for these kinds of values (for readability), but it's pretty clear here so up to you.